src/utils/miscCommands.js

```diff
@@ -0,0 +1,81 @@
+const { EmbedBuilder } = require('@discordjs/builders');
```

Comment:
Is this file only going to be holding help commands? If so, it could be named `helpCommands.js`